src/fontra/views/applicationsettings/applicationsettings.html

```diff
@@ -6,6 +6,7 @@
     <link href="/css/tooltip.css" rel="stylesheet" />
     <title>Fontra Application Settings</title>
     <script type="module" src="/web-components/modal-dialog.js"></script>
+    <script type="module" src="/web-components/grouped-settings.js"></script>
```

Comment:
Please add
```
<script type="module" src="/core/theme-settings.js"></script>
```
after this line, as per #1650.

(While it works without this, given the inclusion of the theme panel, it is good practice and a necessary example for other views.)

Reply:
ok, done.